lib/dish.rb

```diff
@@ -0,0 +1,16 @@
+require "pry"
+
+
+class Dish
+
+  attr_reader :name,
+              :category
+  def initialize(name, category)
+    @name = name
+    @category = {appetizer: "Couscous Salad",
+                entre: "Cocktail Meatballs"
+                      }
```

Comment:
You don't need to do anything special here. You can just set @category to the category value passed in. If you look at the spec, what you put in is exactly what you get out.